ui/src/components/common/Header.js

```diff
@@ -60,7 +66,7 @@ class Header extends React.Component {
       </Toolbar>
     );
 
-    const Header = <PageHeader logo="MOBILE SECURITY SERVICE" logoProps={logoProps} toolbar={toolbar} />;
+    const Header = <PageHeader logo="MOBILE SECURITY SERVICE" onClick={this.onTitleClick} toolbar={toolbar} />;
```

Comment:
Is `onClick` supported by the PageHeader component? (Documentation seems to indicate it isn't - http://patternfly-react.surge.sh/patternfly-4/components/page)

Reply:
I believe it's just a property. I was being passed as a property in the previous implementation also.

Reply:
Can you share where it was being passed previously? 

I haven't ran the code to check if the onClick call works here - do you know if it does?

Reply:
It's in the same file. I removed a few lines above this.

Reply:
Can you confirm the `onClick()` still works, as before it was being passed in as part of the object to the logoProps attribute to PageHeader. PageHeader doesn't seem to accept an `onClick` property[1], and looks like an onClick handler can only be passed in via logoProps to PageHeader

[1]: https://github.com/patternfly/patternfly-react/blob/0b637feaf467a860ee69263bb2c477a3d0090c22/packages/patternfly-4/react-core/src/components/Page/PageHeader.js#L8

Reply:
I can confirm that it's working.

Reply:
Me too, resolving this now